.github/workflows/sanity.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     permissions:
       actions: write   #   to cancel/stop running workflows (styfle/cancel-workflow-action)
       contents: read   #   to fetch code (actions/checkout)
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Same thing needs to be done for sanity-productV2.yml

Reply:
Also, it's a temporary solution I guess? Do you know what is the root cause and what would be the ultimate fix?

Reply:
Ref https://github.com/PrestaShop/PrestaShop/pull/30396